systemd.conf

```diff
@@ -0,0 +1,13 @@
+# shellinabox.service
```

Comment:
Why do we need the systemd file if the entry point is defined in the docker file?